Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/AuditSearchBean.java

```diff
@@ -99,6 +100,12 @@ public AuditSearchBean() {
      */
     public void searchAudit() {
         if (NullChecker.isNullish(messageId) && NullChecker.isNullish(relatesTo)) {
+            if (null != eventEndDate && null != eventEndDate && eventStartDate.after(eventEndDate)) {
+                HelperUtil.addMessageError(null, HelperUtil.readPropertyAdminGui("AuditSearchStartTimeEndTimeMsg",
```

Comment:
since we read from property, just return null if not exist instead of putting msg there